teams/teams-ps/teams/New-Team.md

```diff
@@ -48,7 +48,10 @@ New-Team -GroupId <String> [-Owner <String>] [-AllowGiphy <Boolean>] [-GiphyCont
 ```
 
 ## DESCRIPTION
-Creates a new team with user specified settings, and returns a Group object with a GroupID property.
+Creates a new team with user specified settings, and returns a Group object with a GroupID property. 
+
+> [!WARNING]
+>From version 2.5.0, the return type of this cmdlet has been modified to a resource location for the operation. This resource can be polled for getting the status of the operation using the Get-Operation cmdlet.
```

Comment:
```suggestion
> [!NOTE]
> From Teams version 2.5.0, the return type of this cmdlet has been modified to a resource location for the operation. This resource can be polled to get the status of the operation using the Get-Operation cmdlet.
```

Reply:
@JohanFreelancer9 thanks a lot for the review. Actually, this is the powershell module version, so could we keep it as just version instead of Teams version? Also, this will be a breaking change for admins so we want to show it as a warning than just a note. Please let me know if this would be okay.

Reply:
@shriya999, thanks for your response. Your explanation makes sense, so I have no objection to keeping it as is. I'll delete my suggestion. Thanks again.